forge-gui/res/cardsfolder/t/torrent_of_lava.txt

```diff
@@ -3,8 +3,10 @@ ManaCost:X R R
 Types:Sorcery
 A:SP$ DamageAll | ValidCards$ Creature.withoutFlying | NumDmg$ X | SpellDescription$ CARDNAME deals X damage to each creature without flying. | StackDescription$ SpellDescription
 SVar:X:Count$xPaid
-S:Mode$ Continuous | Affected$ Creature | AddAbility$ TOPrevent | AddSVar$ DBPrevent & DBReplace | EffectZone$ Stack | Description$ As long as Torrent of Lava is on the stack, each creature has "{T}: Prevent the next 1 damage that would be dealt to this creature by CARDNAME this turn."
-SVar:TOPrevent:AB$ Effect | Cost$ T | RememberObjects$ Self | ImprintCards$ OriginalHost | ReplacementEffects$ DBPrevent | ForgetOnMoved$ Battlefield | SpellDescription$ Prevent the next 1 damage that would be dealt to this creature by Torrent of Lava this turn. | StackDescription$ SpellDescription
-SVar:DBPrevent:Event$ DamageDone | ActiveZones$ Command | ValidSource$ Card.IsImprinted | ValidTarget$ Card.IsRemembered | ReplaceWith$ DBReplace | PreventionEffect$ True | Description$ Prevent the next 1 damage that would be dealt to this creature by Torrent of Lava this turn.
-SVar:DBReplace:DB$ ReplaceDamage | Amount$ 1
+S:Mode$ Continuous | Affected$ Creature | AddAbility$ TOPrevent | AddSVar$ DBPrevent & DBReplace | EffectZone$ Stack | Description$ As long as CARDNAME is on the stack, each creature has "{T}: Prevent the next 1 damage that would be dealt to this creature by CARDNAME this turn."
+SVar:TOPrevent:AB$ Effect | Cost$ T | RememberObjects$ Self | ImprintCards$ OriginalHost | ReplacementEffects$ DBPrevent | Duration$ UntilHostLeavesPlayOrEOT | ForgetOnMoved$ Battlefield | SubAbility$ DBCleanup | SpellDescription$ Prevent the next 1 damage that would be dealt to CARDNAME by ORIGINALHOST this turn. | StackDescription$ SpellDescription
+SVar:DBPrevent:Event$ DamageDone | ActiveZones$ Command | ValidSource$ Card.IsImprinted | ValidTarget$ Card.IsRemembered | ReplaceWith$ DBReplace | PreventionEffect$ True | Description$ Prevent the next 1 damage that would be dealt to CARDNAME by ORIGINALHOST this turn.
+SVar:DBReplace:DB$ ReplaceDamage | Amount$ 1 | SubAbility$ ExileEffect
+SVar:ExileEffect:DB$ ChangeZone | Defined$ Self | Origin$ Command | Destination$ Exile
```

Comment:
besides the placeholder changes only adding this exiling should be needed

Reply:
I'd be remiss if I didn't note this peculiar GUI behavior. 
With `{Targeted}`'s wide applicability, I tried to use `{Imprinted}` to no avail. Seems like the reference to Torrent of Lava made on line 7 is displayed properly on Card Detail and ability tooltip but not on the stack item for the activated ability or the log record. As for the command zone effect, at least I figured out I should replace `CARDNAME` with `EFFECTSOURCE` to refer to the creature and not the effect itself.

![lavatorrent_notes](https://github.com/Card-Forge/forge/assets/45150760/bf21480a-4f70-4c73-90c6-1c83d15deb31)

Reply:
yea that's all behaving as expected currently